pkgs/top-level/python-packages.nix

```diff
@@ -6821,6 +6821,10 @@ in {
 
   ckcc-protocol = callPackage ../development/python-modules/ckcc-protocol { };
 
+  bitbox02 = callPackage ../development/python-modules/bitbox02 { };
+
+  noiseprotocol = callPackage ../development/python-modules/noiseprotocol { };
+
```

Comment:
please attempt to sort these with the other packages, don't want to contribute more to the mess